pkg/packages/manager.go

```diff
@@ -32,11 +34,19 @@ type ManagerContext struct {
 	Log           logr.Logger
 }
 
-func (mc *ManagerContext) SetUninstalling(name string) {
+func (mc *ManagerContext) SetUninstalling(namespace string, name string) {
+	mc.Package.Namespace = namespace
 	mc.Package.Name = name
 	mc.Package.Status.State = api.StateUninstalling
 }
 
+func (mc *ManagerContext) getClusterName() string {
```

Comment:
Is this just used for namespaces with a trailing `-` in the namepace, and is that common?

Reply:
see the other comment